src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -319,21 +319,21 @@ FixedwingPositionControl::manual_control_setpoint_poll()
 {
 	_manual_control_setpoint_sub.update(&_manual_control_setpoint);
 
-	_manual_control_setpoint_for_height_rate = _manual_control_setpoint.x;
-	_manual_control_setpoint_for_airspeed = math::constrain(_manual_control_setpoint.z, 0.0f, 1.0f);
+	_manual_control_setpoint_for_height_rate = _manual_control_setpoint.pitch;
+	_manual_control_setpoint_for_airspeed = _manual_control_setpoint.throttle;
 
 	if (_param_fw_pos_stk_conf.get() & STICK_CONFIG_SWAP_STICKS_BIT) {
 		/* Alternate stick allocation (similar concept as for multirotor systems:
 		 * demanding up/down with the throttle stick, and move faster/break with the pitch one.
 		 */
-		_manual_control_setpoint_for_height_rate = -(math::constrain(_manual_control_setpoint.z, 0.0f, 1.0f) * 2.f - 1.f);
-		_manual_control_setpoint_for_airspeed = math::constrain(_manual_control_setpoint.x, -1.0f, 1.0f) / 2.f + 0.5f;
```

Comment:
Not related, but interesting how the previous logic was somehow using the manual control setpoint concept for it's own airspeed setpoint :thinking:, seems like an unnecessary conceptualization, but doesn't affect the PR.